src/libraries/Microsoft.PowerFx.Core/Utils/TextFormatUtils.cs

```diff
@@ -120,7 +120,7 @@ public static bool IsValidFormatArg(string formatString, CultureInfo formatCultu
                     mCount = 0;
                 }
 
-                if (formatStr[i] == 'a' || formatStr[i] == 'A')
+                if ((formatStr[i] == 'a' || formatStr[i] == 'A') && (i == 0 || formatStr[i - 1] != '\''))
```

Comment:
>(i == 0 || formatStr[i - 1] != '\'' [](http://example.com/codeflow?start=68&length=35)

Is this change related to thousands grouping character and escaping?

Reply:
This is not issue of thousand grouping but when i test it and i found that I only want to do the am pm restrict when it is not in escaping ''. so i just fix here.